src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,17 @@
-export const UserInfo = () => {};
+import { User } from '../../types';
+
+type UserInfoProps = {
+  user?: User,
```

Comment:
your user should always accept user not null. check for null in parent component. and don't make this prop optional since it is required